client/src/components/Ticketing/cart/Cart.tsx

```diff
@@ -70,6 +88,29 @@ const Cart = () => {
     handleClick2();
   };
 
+  const printDiscountText = (disc: DiscountItem) => {
+    if (disc.code === '' ) return;
+
+    if (disc.amount === 0) {
+      return (disc.percent + '% discount');
+    } else {
+      return ('$' + disc.amount + ' discount');
+    }
+  };
+
+  const applyDiscount = (e: React.FormEvent) => {
```

Comment:
The `e` parameter doesn't look like it's getting used, can it be removed?

Reply:
Yes, it's gone.